src/Command/RunCommand.php

```diff
@@ -110,7 +111,13 @@ protected function configure(): void
                 'no-progress',
                 null,
                 InputOption::VALUE_NONE,
-                'Do not output progress bars'
+                'Do not output progress bars and mutation count during progress. Automatically enabled if a CI is detected'
+            )
+            ->addOption(
+                'progress',
```

Comment:
what about `--with-progress`?

```
--no-progress
--with-progress
```

but the current one is also ok, just another idea :)

Reply:
`force-progress`? It's more in line with the description and if we have `with-progress` I would think of `with-no-progress` or `without-progress` rather than `no-progress`

Reply:
`--force-progress` 👌